mcs/class/System.Web.Services/System.Web.Services/WebMethodAttribute.cs

```diff
@@ -39,7 +39,10 @@ public enum TransactionOption {Disabled , NotSupported , Required , RequiresNew
 #endif
 
 	[AttributeUsage(AttributeTargets.Method, Inherited = true)]
-	public sealed class WebMethodAttribute : Attribute {
+#if !XAMMAC_4_5
```

Comment:
Because Sebastien asked me to mostly. We include it in our mobile file list, so I didn't want to rip it out the whole file (and whatever that entails). 

From: https://msdn.microsoft.com/en-us/library/system.web.services.webmethodattribute(v=vs.110).aspx

Adding this attribute to a method within an XML Web service created using ASP.NET makes the method callable from remote Web clients.

I can undo this if desired.